views/shared/header/menu.jade

```diff
@@ -2,7 +2,7 @@ nav.toolbar(ng-controller='AuthCtrl')
   ul.toolbar-mobile-nav
     li.toolbar-mobile
       a(ng-click='expandMenu("mobile")', ng-class='{active: _expandedMenu=="mobile"}')
-        span  &#9776;
+        span.glyphicon.glyphicon-th-list
```

Comment:
This instance of `&#9776;` has been replaced by glyphicon-align-justify in https://github.com/HabitRPG/habitrpg/pull/4855

I'm not sure if the other instances below need to be replaced for the sake of mobile devices (except maybe tablets?). See discussion in https://github.com/HabitRPG/habitrpg/issues/3411